shopify/resources/order.py

```diff
@@ -19,3 +20,16 @@ def transactions(self):
 
     def capture(self, amount=""):
         return Transaction.create({"amount": amount, "kind": "capture", "order_id": self.id})
+
+    def add_fulfillment(self, fulfillment):
+        if not self.id:
+            self.save()
```

Comment:
That makes sense, if not at least for consistency. I'll submit the modification soon.